tools/flaky-test-reporter/main.go

```diff
@@ -26,15 +26,22 @@ import (
 	"os"
 
 	"github.com/knative/test-infra/shared/prow"
+	ftrConfig "github.com/knative/test-infra/tools/flaky-test-reporter/config"
 )
 
 func main() {
 	serviceAccount := flag.String("service-account", os.Getenv("GOOGLE_APPLICATION_CREDENTIALS"), "JSON key file for GCS service account")
 	githubAccount := flag.String("github-account", "", "Token file for Github authentication")
 	slackAccount := flag.String("slack-account", "", "slack secret file for authenticating with Slack")
+	configPath := flag.String("configfile", "", "Config file for overriding default config file")
 	dryrun := flag.Bool("dry-run", false, "dry run switch")
 	flag.Parse()
 
+	config, err := ftrConfig.NewConfig(*configPath)
+	if nil != err {
+		log.Fatalf("config file '%s' not found '%v'", *configPath, err)
```

Comment:
Re-word to config cannot be created? There can be other failure reasons than not found

Reply:
good idea, done